atomicsabi64/atomicsabi64.rst

```diff
@@ -0,0 +1,1252 @@
+..
+   Copyright (c) 2024, Arm Limited and its affiliates.  All rights reserved.
+   CC-BY-SA-4.0 AND Apache-Patent-License
+   See LICENSE file for details
+
+.. |release| replace:: 2024Q1
+.. |date-of-issue| replace:: 5\ :sup:`th` July 2024
+.. |copyright-date| replace:: 2024
+.. |footer| replace:: Copyright © |copyright-date|, Arm Limited and its
+                      affiliates. All rights reserved.
+
+.. _ARMARM: https://developer.arm.com/documentation/ddi0487/latest
+.. _AAELF64: https://github.com/ARM-software/abi-aa/releases
+.. _CPPABI64: https://github.com/ARM-software/abi-aa/releases
+.. _CSTD: https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1548.pdf
+.. _PAPER: https://doi.org/10.1109/CGO57630.2024.10444836
+
+*********************************************************************************************
+C/C++ Atomics Application Binary Interface Standard for the Arm\ :sup:`®` 64-bit Architecture
+*********************************************************************************************
+
+.. class:: version
+
+|release|
+
+.. class:: issued
+
+Date of Issue: |date-of-issue|
+
+.. class:: logo
+
+.. image:: Arm_logo_blue_RGB.svg
+   :scale: 30%
+
+.. section-numbering::
+
+.. raw:: pdf
+
+   PageBreak oneColumn
+
+
+Preamble
+========
+
+Abstract
+--------
+
+This document describes the C/C++ Atomics Application Binary Interface for the
+Arm 64-bit architecture. This document concerns the valid Mappings from C/C++
+Atomic Operations to sequences of A64 instructions. For matters concerning the
+memory model, please consult §B2 of the Arm Architecture Reference Manual
```

Comment:
`For further information on the memory model, refer to the Arm......`

Keep sentences as simple as possible.